test/javascript/components/concept-map/ConceptMap.test.tsx

```diff
@@ -100,53 +96,39 @@ describe('<ConceptMap />', () => {
         },
       ],
       status: { test1: 'mastered', test2: 'available' },
-      exerciseStatuses: {
-        testA: ['complete'],
-        testB: ['available'],
+      exercisesData: {
+        testA: [{ status: 'completed' }],
+        testB: [{ status: 'available' }],
       },
     }
 
-    const renderedConceptMap = renderConceptMap(config)
-    await waitForConceptMapReady(renderedConceptMap, config)
+    await renderConceptMap(config)
 
     expect(screen.queryByText('Test A')).toBeInTheDocument()
     expect(screen.queryByText('Test B')).toBeInTheDocument()
     expect(screen.queryByTestId('path-test-a-test-b')).toBeInTheDocument()
   })
 })
 
-const renderConceptMap = (config: IConceptMap) =>
-  render(
-    <ConceptMap
-      concepts={config.concepts}
-      levels={config.levels}
-      connections={config.connections}
-      status={config.status}
-      exerciseStatuses={config.exerciseStatuses}
-    />
-  )
-
-const waitForConceptMapReady = async (
-  renderedConceptMap: ReturnType<typeof renderConceptMap>,
-  config: IConceptMap
-) => {
-  await Promise.all(
-    config.concepts
-      .map((concept) => concept.name)
-      .map((conceptName) =>
-        waitFor(() =>
-          expect(renderedConceptMap.getByText(conceptName)).toBeInTheDocument()
-        )
-      )
-  )
-}
```

Comment:
@neenjaw By wrapping the `renderConceptMap` in `async act`, it seems like this bit above can be removed. I got here because I was getting loads of warnings about nested `act` before I did this. Let me know if there's something I've overlooked here pls :)